interface/javascript/main.js

```diff
@@ -750,3 +762,43 @@ function updateMap(mymap) {
                 return coords.map(i => i.x + ',' + i.y).join(' ');
             });
 }
+
+function openWeatherMapo(mymap, layer) {
+  return L.tileLayer('https://tile.openweathermap.org/map/{layer}/{z}/{x}/{y}.png?appid={api_key}', {
+   layer: layer,
+   maxZoom:17,
+   api_key: '2733c9a9c041a4ba7ce1963ae1a97dd4'
+   })
+}
+
+function heatMapMaker(mymap, cacheData, mode) {
+  // console.log(cacheData[0])
+
+  if (mode === "label") {
+    var heatCoordinates = cacheData.map(x => [x.feature.geometry.coordinates[0][0][0][1],x.feature.geometry.coordinates[0][0][0][0], x.label])
```

Comment:
Is it possible to shorten these line lengths a bit?

Reply:
done